crates/ruff_cli/src/commands/format.rs

```diff
@@ -515,7 +515,7 @@ impl<'a> FormatResults<'a> {
         if changed > 0 && unchanged > 0 {
             writeln!(
                 f,
-                "{} file{} {}, {} file{} left unchanged",
+                "{} file{} {}, {} file{} already formatted",
```

Comment:
I think we want to apply this change only in the event that we have `FormatMode::Check | FormatMode::Diff` -- do you mind tweaking, similar to "would be reformatted" below?

Reply:
Sorry, I don't understand: _Check and Diff_ here are like the lower branch of the 'if'

Reply:
@VictorGob - I pushed an update to illustrate what I meant. What do you think of this approach?

Reply:
**Chef kiss**